docs/advanced-usage/enabling-natural-language-search.md

```diff
@@ -0,0 +1,129 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Enabling Natural Language Search
```

Comment:
Not sure if `enabling` is the right verb here, as this wouldn't be a one time toggle, but I also wanted this to feel more approachable. Thoughts?

Reply:
How about `How to Set Up Natural Language Search` or `Natural Language Search Setup`?